src/views/Proposals.vue

```diff
@@ -28,21 +28,30 @@
     <Container :slim="true">
       <Block :slim="true">
         <div class="px-4 py-3 bg-gray-dark overflow-auto menu-tabs">
+          <a
+            :key="'Core'"
+            :href="'#/'+key+'/#Core'"
```

Comment:
You shouldn't use `<a>` when you stay on the site but use `<router-link>` instead. Also we don't want to have 2 hashes in the url and it's better to use template strings like this \`${key}/core`

Reply:
The `<a>` was already used, so not sure why it was while it could have been indeed router-link, but also here isnt a router link at all, this is just on the page, so we can use `<a>` and for the hash. Also sure we can move it to full url, then we can do also what is right for the routerlink etc., lets try deploy this now then we add these updates after today